api.js

```diff
@@ -9,13 +9,29 @@ const getArticles = () => {
     .then(({data}) => {
         return data.articles
     })
+    .catch((err) => {
+        console.log(err)
+    })
 }
 
 const getArticlesById = (article_id) => {
     return api.get(`/articles/${article_id}`)
     .then(({data}) => {
         return data.article
     })
+    .catch((err) => {
+        console.log(err)
+    })
+}
+
+const getArticleComments = (article_id) => {
+    return api.get(`/articles/${article_id}/comments`)
+    .then(({data}) => {
+        return data.comments
+    })
```

Comment:
We should have a single catch block per promise chain which should be at the end of the chain. These catch blocks in this file should be removed